Sources/StytchCore/StytchClientCommon/StytchClientType.swift

```diff
@@ -9,14 +9,30 @@ protocol StytchClientType {
 }
 
 extension StytchClientType {
+    private static var keychainClient: KeychainClient { Current.keychainClient }
```

Comment:
I'm unable to use propertyWrappers in extensions, and this is a protocol extension (no implementations allowed in the protocol definition), so this is a workaround to be somewhat similar to the property wrapper situation. I _potentially could_ add these dependencies as requirements to the protocol itself, and then use those in this extension, but would have to declare the implementations for the dependencies in each of the StytchClientType implementations. Perhaps will move to this in the future.